package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "agroptima-design-system",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "scripts": {
     "dev": "npm run storybook",
     "storybook": "storybook dev -p 6006 --ci",
     "build": "npm run build-storybook",
     "build-storybook": "storybook build",
     "lint": "eslint",
     "lint:fix": "eslint src --fix",
-    "publish-chromatic": "npx chromatic --exit-zero-on-changes"
+    "publish-chromatic": "npx chromatic --exit-zero-on-changes",
+    "chromatic": "npx chromatic --exit-zero-on-changes"
```

Comment:
Better not to duplicate, if we want to use an alias we can do as in the build and build-storybook, launch the command by npm.

Reply:
I forgot to ask about this. I remember that we changed the "chromatic" line by the "publish-chromatic" line, but magically both lines appeared on the package. I can safely erase the "chromatic" line, right? I mean, nobody and no process is using `chromatic` command.